Client/Frontend/Browser/HomePageHelper.swift

```diff
@@ -73,4 +73,41 @@ class HomePageHelper {
             })
         navigationController?.presentViewController(alertController, animated: true, completion: nil)
     }
+}
+
+/// Accessors for homepage details from the app state.
+/// These are pure functions, so it's quite ok to have them 
+/// as static.
+class HomePageAccessors {
+    private static let getPrefs = Accessors.getPrefs
```

Comment:
not sure if I understand why we are making a reference to `getPrefs` here as a variable. For me it doesn't add any clarity to the usage of it here. `HomePageAccessors` could simply inherit from `Accessors` and that would serve the same purpose (or am I thinking too OO here?)